cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -27,8 +27,8 @@ func extractExecutionState(dir string, targetHash flow.StateCommitment, outputDi
 	filePath := path.Join(outputDir, "root.checkpoint")
 
 	newState, err := led.ExportCheckpointAt(targetHash,
-		[]ledger.Migration{migrations.MultipleContractMigration, migrations.AddMissingKeysMigration},
-		[]ledger.Reporter{},
+		[]ledger.Migration{migrations.StorageFeesMigration},
```

Comment:
you also need to update this line: 
led, err := complete.NewLedger(dir, complete.DefaultCacheSize, &metrics.NoopCollector{}, log, nil, 0)
to 
led, err := complete.NewLedger(dir, complete.DefaultCacheSize, &metrics.NoopCollector{}, log, nil, complete.DefaultPathFinderVersion)